### PR TITLE
Vd starting on UI

### PR DESCRIPTION
**Pull request by viviancreates**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
```
Comment (aldoportillo) [inline app/controllers/application_controller.rb:2]: good job force authenticating user

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -6,13 +6,27 @@
...
+      <% end %>
+      
+      <% if alert.present? %>
+        <%= render partial: "shared/flash", locals: { message: alert, css_class: "danger" } %>
+      <% end %>
+
+     <div class="container">
+      <%= yield %>
+    </div>
+
```
Comment (aldoportillo) [inline app/views/layouts/application.html.erb:29]: you're missing a `</div>` here. also why do you have two `<div class="container">`

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -0,0 +1,52 @@
+<nav class="navbar navbar-expand-lg bg-light mb-4">
+  <div class="container">
+
+    <!-- <a class="navbar-brand" href="#">Navbar</a> -->
```
Comment (aldoportillo) [inline app/views/shared/_navbar.html.erb]: remove commented code

File: config/routes.rb [hunk 4]
```diff
@@ -1,10 +1,13 @@
 Rails.application.routes.draw do
-  resources :likes
-  resources :follow_requests
+  root "photos#index"
```
Comment (aldoportillo) [inline config/routes.rb:2]: nice

File: lib/tasks/dev.rake [hunk 5]
```diff
@@ -2,12 +2,28 @@ desc "Fill the database tables with some sample data"
...
+  starting = Time.now
+
   if Rails.env.development?
-    FollowRequest.destroy_all
-    Comment.destroy_all
-    Like.destroy_all
-    Photo.destroy_all
-    User.destroy_all
+    FollowRequest.delete_all
+    Comment.delete_all
```
Comment (aldoportillo) [inline lib/tasks/dev.rake:9]: good job

**Review by aldoportillo — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (Ritumka) [reply app/views/layouts/application.html.erb:29]: Yep, the last div class="container" should be just deleted.

File: app/views/photos/index.html.erb
Comment (Ritumka) [inline app/views/photos/index.html.erb:1]: #3  I like that now we can collect all our alert messages at the same partial file, so we don't need to type full code in an every view file we create.

File: app/views/shared/_flash.html.erb
Comment (Ritumka) [inline app/views/shared/_flash.html.erb:1]: Super convenient, isn't it?

**Review by Ritumka — COMMENTED** [review]
Great job!

**Pull request merged**